installer/win/Brackets_en-us.wxl

```diff
@@ -54,5 +54,8 @@
     <String Id="ExitDialogTitle" Overridable="yes">{\WixUI_Font_BannerTitle}Completed the [ProductName] Setup Wizard</String>
     <String Id="VerifyReadyDlgInstallText" Overridable="yes">Click Install to begin, or Cancel to exit.</String>
     <String Id="VerifyReadyDlgRemoveText" Overridable="yes">Click Remove to uninstall !(loc.ShortProductName), or Cancel to exit.</String>
-    <String Id="OpenWithText" Overridable="yes">Open With !(loc.ShortProductName)</String>
-</WixLocalization>
\ No newline at end of file
+    <String Id="ExplorerFileOpenContextMenu" Overridable="yes">Open with !(loc.ShortProductName)</String>
+    <String Id="ExplorerFolderOpenContextMenu" Overridable="yes">Open as !(loc.ShortProductName) project</String>
+    <String Id="AddBracketsToPath" Overridable="yes">Add &quot;!(loc.smallProductName)&quot; launcher to PATH for command line use</String>
```

Comment:
The user does not need to know the exact case of the launcher (Windows is case-insensitive, after all) nor that we use a launcher at all. This makes it feel a bit technical. How about "Add !(loc.smallProductName) to the system PATH for command line launching"?